src/components/List/ListItem/ListItem.jsx

```diff
@@ -167,9 +167,11 @@ const ListItem = ({
       </div>
     ) : null;
 
+  const hasRowActions =
```

Comment:
nitty gritty, but can't this just be `typeof rowActions === 'function' || rowActions?.length`?